.vscode/extensions.json

```diff
@@ -3,6 +3,7 @@
     // for the documentation about the extensions.json format
     "recommendations": [
         "ms-vscode.PowerShell",
-        "ms-vscode.csharp"
+        "ms-vscode.csharp",
```

Comment:
@bergmeister I didn't think that json allowed for comments. Is this specific to vscode?

Reply:
There is `JSONC`, which is basically JSON with comments and VS Code supports it. Those comments are from the default template, which are being left there as they are quite useful